src/components/Grid.tsx

```diff
@@ -34,7 +34,7 @@ export const Grid: React.FunctionComponent<GridProps> = ({
   rowGap = rowGap || gap
 
   return (
-    <section
+    <div
```

Comment:
This is more semantically correct right? `section` refers more to a "section" in a document rather than this..being a grid, which is more of a container. Am I understanding this properly?

Reply:
I had the same thought initially, but in practice I noticed a couple of related things that changed my mind.
1. In some cases the Grid belongs logically grouped with some other content, e.g. a header, or supplemental text. In those cases, the semantic "section" is all of that content taken together, not _just_ the grid.
2. In some cases, the component using the grid already had a higher level notion of _section_, which resulted in nested sections. Nested sections aren't prohibited, but semantically it was again incorrect, since the section is meant to map to a section of the page that would be shown in a document outline.